### PR TITLE
Add juju-rrun plugin to pretty print 'juju run' output

### PR DESCRIPTION
**Pull request by freyes**
This plugin makes easier to read the stdout/stderr when running commands
in more than one unit and interpreting new line characters (`\n`), this
is ideal for verbose commands like `crm_mon -1`.

Example output of `sudo service nova-api-os-compute status`:

$ juju-rrun --service nova-cloud-controller 'sudo service nova-api-os-compute status'
---------- 5 nova-cloud-controller/0
---------- stdout ----------
nova-api-os-compute start/running, process 26325

---------- 6 nova-cloud-controller/1
---------- stdout ----------
nova-api-os-compute start/running, process 29767

---------- 7 nova-cloud-controller/2
---------- stdout ----------
nova-api-os-compute start/running, process 29691

### CODE REVIEW

**Comment by marcoceppi** [comment]
This is fantastic, thank you for the submission. Unfortunately, it does not meet the plugin guidelines, mainly:

> All plugins must be able to handle and respond to a --help flag and a --description flag. These are used by juju when a user runs juju help plugins and when a user runs juju help <plugin> they call the --description and --help flags respectively.

Make sure the plugin can handle these two flags, where one prints out a standard help output and the second prints out a brief one line description. Both should terminate execution of the script after printing it's information.

**Comment by freyes** [comment]
@marcoceppi thanks for your feedback, I just pushed a new version of the patch to address your comments.

Thanks

File: juju-rrun [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+from __future__ import print_function
+import json
+import subprocess
+import sys
+
+separator = '----------'
+
+
+def print_help():
+    print('Pretty print of juju run output')
```
Comment (marcoceppi) [inline juju-rrun]: You have these backwards. print_help should be what print_description has and vice versa. Description is oneline help is help output.

**Comment by freyes** [comment]
This is already addressed by juju 1.24, so I'm closing this pull request.

Thanks,

**Pull request closed without merging**